03.bowling/bowling.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'rubygems'
+
+obtained_score = ARGV[0]
+scores = obtained_score.split(',')
+shots = []
+
+scores.each do |score|
+  if score == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << score.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+point = 0.upto(9).sum do |i|
+  current_frame = frames[i]
+  next_frame = frames[i + 1]
+  after_next_frame = frames[i + 2]
```

Comment:
```suggestion
  current_frame, next_frame, after_next_frame = frames[i, 3]
```

みたいに書くこともできますね（好みの問題）

Reply:
なるほど、覚えておきますありがとうございます！